doc/rst/usingchapel/executing.rst

```diff
@@ -91,21 +95,77 @@ or:
 The compiler-established default can still be overridden when
 executing the program, as shown above.
 
+~~~~~~~~~~~~~~~~~~
+Configuration File
+~~~~~~~~~~~~~~~~~~
+
 Configuration values can also be passed to a Chapel program through a
-configuration file, specified by the execution time ``-f`` option. These
-configuration files can contain a whitespace- or newline-separated list of
-configuration assignments. Comments are supported with the ``#`` character.
+configuration file, specified by the execution time ``-f`` option. The format
+is described in the example below.
 
-For example:
 
-    .. code-block:: sh
+Consider the following program:
+
+   .. code-block:: chapel
+
+       // program.chpl
+       config const msg: string,
+                    val1: real,
+                    val2: real;
+
+
+       proc main() {
+         writeln(msg);
+         writeln(val1);
+         writeln(val2);
+       }
+
+The above program can have its configuration variables defined by this
+configuration file:
+
+    .. code-block:: python
+
+        # program.input
+
+        msg="hello world"
+        val1=1.61803
+        val2=3.14159
+
+Configuration files can contain a whitespace- or newline-separated list of
+configuration assignments and comments are supported with the ``#`` character.
+The configuration file above can also be written like this:
 
-        # hello2-module.input
-        message="Hello from a Chapel configuration file"
+    .. code-block:: python
+
+        # program.input
+
+        val1=1.61803 val2=3.14159
+        msg="hello world" # This is a comment
+
+
+The ``program.input`` is passed during execution with the ``-f`` flag:
 
     .. code-block:: sh
 
-        ./hello2-module -fhello2-module.input
+        # config variables are populated by program.input values
+        ./program -fprogram.input
+
+
+.. warning::
+    Assignments cannot contain whitespaces outside of quotes, so the following
```

Comment:
`chplspell` doesn't like the word "whitespaces" - maybe drop the 's' from the end?

Reply:
FWIW, I've taken the tack of adding to the chplspell dictionary any word that's natural to use in the project.  As long as it's spelled right. 

Including "words" that are formed by command line option, as "fhello" for the "-fhello" removed below.  :(